packages/odyssey/CHANGELOG.md

```diff
@@ -5,6 +5,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.0] - 2020-07-31
+
+### Added
+
+- `@okta/odyssey-icons` package: includes an initial set of SVGs and process for batch optimizing via `svgo`
+- `@okta/vuepress-theme-nimatron` package: supplies the default theme for Odyssey's docs
+
+### Changed
+
+- [Breaking Change] Changed the Odyssey documentation platform from `hexo` to `vuepress`
```

Comment:
I wouldn't call this out as a `[Breaking Change]`. This is just internal tooling used by our development team. Consumers of Odyssey **will not need to make updates** to their apps based on this change.